contracts/src/v0.8/ccip/rmn/RMNHome.sol

```diff
@@ -81,7 +81,7 @@ contract RMNHome is Ownable2StepMsgSender, ITypeAndVersion {
 
   struct SourceChain {
     uint64 chainSelector; // ─╮ The Source chain selector.
-    uint64 f; // ─────────────╯ Maximum number of faulty observers; f+1 observers required to agree on an observation for this source chain.
+    uint64 fObserve; // ─────────────╯ Maximum number of faulty observers; f+1 observers required to agree on an observation for this source chain.
```

Comment:
pls align the struct packing comments

Reply:
hm i wonder how that got messed up. Good catch.